packages/react-components/src/select/src/Select.tsx

```diff
@@ -237,7 +237,8 @@ export function InnerSelect(props: InnerSelectProps) {
                             fluid && "fluid",
                             active && "active",
                             focus && "focus",
-                            hover && "hover"
+                            hover && "hover",
+                            isNil(selectedItem) && "placeholder"
```

Comment:
You should use "has-placeholder" instead which is more in line with what we use elsewhere for this.